draft-thomson-webpush-protocol.xml

```diff
@@ -561,20 +492,18 @@ DATA         [stream 4] +END_STREAM
       </t>
 
       <t>
-        A user agent can request the contents of the "registration" resource
+        A user agent can request the contents of the "monitor" resource
         immediately by including a <xref target="RFC7240">Prefer header
         field</xref> with a "wait" parameter set to "0".  The push server SHOULD
-        return a link reference to the "registration" resource and expiration
+        return a link reference to the "monitor" resource and expiration
```

Comment:
I think that this might have been in error originally.  Returning a link to the "push" resource would be useful.  No sense in providing a self-pointer.